client/src/components/EpisodeCard.spec.js

```diff
@@ -4,38 +4,62 @@ import EpisodeCard from './EpisodeCard'
 describe('EpisodeCard', () => {
   it('renders', () => {
     const episode = {
+      id: 123,
       name: 'Title',
       overview: 'Lorem Ipsum Dolor',
       episode_number: 1,
     }
```

Comment:
Since you are using the same object in each test, I would define it once before the `describe`. Also, your tests are quite generic and could well be one test:
```js
it('renders a card as listitem with correct text', () ....
```
Or something like that.